src/module/rolls/chat-message.ts

```diff
@@ -141,8 +157,7 @@ export class IronswornRollChatMessage {
     // FIXME: oof. so, "roll" is a tricky word since in english it can function as a verb or a noun. it gets even more complicated when you introduce grammatical gender and word order.
```

Comment:
If this bothers you, we can introduce a new i18n key.

```json
"RollPlusStat": "Roll +{stat}"
```

Which you can invoke using format:

```ts
game.i18n.format('IRONSWORN.RollPlusStat', {stat: plusStat})
```